packages/camera/camera_avfoundation/example/ios/RunnerTests/AvailableCamerasTests.swift

```diff
@@ -7,6 +7,10 @@ import XCTest
 
 @testable import camera_avfoundation
 
+#if canImport(camera_avfoundation_objc)
```

Comment:
why is this check required and when will it be false?

Reply:
It will be false when cocoapods are used. From the pods perspective, there is still a single `camera_avfoundation` module containing both swift and objc code

Reply:
Of course, this is temporary once everything is migrated to Swift we will have a single SwiftPM target and those conditional imports will no longer be needed